package.json

```diff
@@ -120,5 +121,8 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx,css,md}": "prettier --write"
+  },
```

Comment:
please move it to dev deps (packaging requirements)

Reply:
oops, fixed